tests/release/pipelines/push_to_external_registry.go

```diff
@@ -30,7 +30,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-var _ = framework.ReleasePipelinesSuiteDescribe("[HACBS-1571]test-release-e2e-push-image-to-pyxis", Label("release-pipelines", "pushPyxis", "HACBS"), func() {
+var _ = framework.ReleasePipelinesSuiteDescribe("[HACBS-1571]test-release-e2e-push-image-to-pyxis", Label("release-pipelines", "pushPyxis", "HACBS"), Pending, func() {
```

Comment:
The variables "PYXIS_STAGE_CERT/KEY" are already created in the vault.  Would you like to try it before set it as pending?

Reply:
No, for this PR I'd just like to turn the tests off. We have tickets for re-enabling the tests and the JIRA for re-enabling the pyxis test can be used for that investigation